posix.mak

```diff
@@ -472,6 +472,15 @@ docs-prerelease.json : ${DMD} ${DRUNTIME_DIR} \
 	  --only-documented $(MOD_EXCLUDES_RELEASE)
 	rm .prerelease-files.txt .prerelease-dummy.html
 
+${GENERATED}/upcoming.dd: | ${GENERATED}
+	$(STABLE_RDMD) ../tools/build_changelog.d ../phobos/changelog --without-macros > $@
+	$(STABLE_RDMD) ../tools/changed.d v2.071.0..upstream/master -o ${GENERATED}/changlog.tmp
```

Comment:
it would be nicer if the `changed.d` could output to stdout, but this should work too ;-)